packages/forms-web-app/src/sass/components/ui/file-upload.scss

```diff
@@ -0,0 +1,10 @@
+#file-upload {
```

Comment:
Might be good to give the elements more specific class names and maybe even put it into the component just to reduce risk of any CSS contamination.